src/Module.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\I18n\PhoneNumber;
+
+final class Module
```

Comment:
We could drop a line here, explaining that this is for compatibility with the module manager?